### PR TITLE
[Profile] Edit paymento info

### PR DESCRIPTION
**Pull request by lemefe21**

### CODE REVIEW

File: design/src/main/java/com/ingresse/design/ui/editText/DSEditText.kt [hunk 1]
```diff
@@ -266,4 +266,9 @@ class DSEditText(context: Context, attrs: AttributeSet): FrameLayout(context, at
         txt_hint.setTextColor(hintColor)
         isWrong = false
     }
+
+    fun validateEmptyEditTextError() {
+        if(edit_text.text.count() == 0) { setEditTextError() }
```
Comment (PhillipiLino) [inline design/src/main/java/com/ingresse/design/ui/editText/DSEditText.kt]: Use `isEmpty()` to check if string is empty

File: design/src/main/java/com/ingresse/design/ui/spinner/DSSpinner.kt [hunk 2]
```diff
@@ -75,4 +77,21 @@ class DSSpinner(context: Context, attrs: AttributeSet): FrameLayout(context, att
...
 
     private fun createAdapter() = DSSpinnerAdapter(context)
+
+    fun setSpinnerHintTextDefault() {
+        txt_hint.setTextColor(hintColor)
+        isWrong = false
+    }
+
+    fun setSpinnerHintTextError() {
+        spinner.setSelection(0)
```
Comment (PhillipiLino) [inline design/src/main/java/com/ingresse/design/ui/spinner/DSSpinner.kt]: Why we set first item when show error??

File: design/src/main/res/values/dimens.xml [hunk 3]
```diff
@@ -31,6 +31,7 @@
     <dimen name="font_very_small">12sp</dimen>
 
     <!-- SPACING -->
+    <dimen name="spacing_x0">0dp</dimen>
```
Comment (PhillipiLino) [inline design/src/main/res/values/dimens.xml]: don't need this, just not use space

File: design/src/main/java/com/ingresse/design/ui/editText/DSEditText.kt [hunk 4]
```diff
@@ -100,7 +100,12 @@ class DSEditText(context: Context, attrs: AttributeSet): FrameLayout(context, at
         editText.setSelection(0)
     }
 
-    fun setTextDS(txt: String?) {
+    fun setTextDS(txt: String?, cleanWhenEmpty: Boolean = false, wrongWhenEmpty: Boolean = false) {
+        if (cleanWhenEmpty) {
+            editText.text.clear()
+            if (!editText.hasFocus()) animateHintToCenter()
+        }
+        if (wrongWhenEmpty) setEditTextError()
```
Comment (tmikulsk1) [inline design/src/main/java/com/ingresse/design/ui/editText/DSEditText.kt]: Here you need to ask if field is empty too.

File: design/src/main/java/com/ingresse/design/ui/editText/DSEditText.kt [hunk 5]
```diff
@@ -100,7 +100,12 @@ class DSEditText(context: Context, attrs: AttributeSet): FrameLayout(context, at
         editText.setSelection(0)
     }
 
-    fun setTextDS(txt: String?) {
+    fun setTextDS(txt: String?, cleanWhenEmpty: Boolean = false, wrongWhenEmpty: Boolean = false) {
+        if (cleanWhenEmpty) {
```
Comment (tmikulsk1) [inline design/src/main/java/com/ingresse/design/ui/editText/DSEditText.kt]: Here you need to ask if field is empty.

**Pull request closed without merging**